src/modules/top-level.nix

```diff
@@ -104,7 +104,18 @@ in
 
   config = {
     # TODO: figure out how to get relative path without impure mode
-    env.DEVENV_ROOT = builtins.getEnv "PWD";
+    env.DEVENV_ROOT =
+      let
+        pwd = builtins.getEnv "PWD";
+      in
+      if pwd == null then
```

Comment:
```suggestion
      if pwd == "" then
```

Reply:
Good call. Fixed this. Also found a way to test this properly, which also catches this :+1: